github-orgs/kubeflow/org.yaml

```diff
@@ -966,7 +966,7 @@ orgs:
           release-managers:
             description: Kubeflow Release Team - Managers
             members:
-              - DnPlas
+              - rimolive
```

Comment:
Just want to make sure @DnPlas comment on the handover. And then I can approve.

Reply:
+1, @rimolive will be the next release manager.